### PR TITLE
Paymaya icon

### PR DESCRIPTION
**Pull request by jiroquijano**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by tauthomas01** [comment]
Hi @jiroquijano 

Please follow the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) guideline:
- The background is missing
- The border of the icon is missing

You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.

File: app/assets/images/payment_icons/paymaya.svg [hunk 1]
```diff
@@ -0,0 +1,2 @@
... d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" opacity=".05"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/><g transform="translate(4 1)" fill-rule="evenodd" stroke="null"><path fill="#167eff" d="m22.446,15.96906a0.436,0.436 0 0 0 -0.38,-0.3c-0.488,-0.041 -0.98,-0.081 -1.467,-0.133a20.579,20.579 0 0 1 -5.19,-1.171a9.788,9.788 0 0 1 -5.212,-4.156a6.694,6.694 0 0 1 -0.689,-6.108c0.094,-0.234 0.3,-0.58 0.227,-0.686s-0.184,-0.074 -0.276,-0.067a7.887,7.887 0 0 0 -2.38,0.533a6.639,6.639 0 0 0 -4.25,6.229a7.787,7.787 0 0 0 1.3,4.051a11.252,11.252 0 0 0 4.343,3.908a21.9,21.9 0 0 0 6.52,2.164a63.014,63.014 0 0 0 7.6,0.584c0.336,0.011 0.49,0 0.748,0c0.344,-0.034 0.445,-0.274 0.39,-0.578s-0.175,-0.683 -0.307,-1.166a50.109,50.109 0 0 0 -0.977,-3.104z"/><path fill="#8dc540" d="m26.022,13.82306c-0.022,-0.1 -0.122,-0.552 -0.223,-0.917a27.341,27.341 0 0 0 -2.94,-7.114a12.05,12.05 0 0 0 -3.3,-3.668a5.993,5.993 0 0 0 -4.932,-1.16a5.388,5.388 0 0 0 -4.194,4a5.6,5.6 0 0 0 0.422,4.031a8.2,8.2 0 0 0 4.094,3.923a17.754,17.754 0 0 0 6.561,1.525a38.759,38.759 0 0 0 4.1,0.018a0.48,0.48 0 0 0 0.412,-0.643"/></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/paymaya.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" aria-labelledby="pi-paymaya" role="img" viewBox="0 0 38 24"><title id="pi-paymaya">Paymaya</title><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" opacity=".07"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/><g transform="translate(4 1)" fill-rule="evenodd" stroke="null"><path fill="#167eff" d="m22.446,15.96906a0.436,0.436 0 0 0 -0.38,-0.3c-0.488,-0.041 -0.98,-0.081 -1.467,-0.133a20.579,20.579 0 0 1 -5.19,-1.171a9.788,9.788 0 0 1 -5.212,-4.156a6.694,6.694 0 0 1 -0.689,-6.108c0.094,-0.234 0.3,-0.58 0.227,-0.686s-0.184,-0.074 -0.276,-0.067a7.887,7.887 0 0 0 -2.38,0.533a6.639,6.639 0 0 0 -4.25,6.229a7.787,7.787 0 0 0 1.3,4.051a11.252,11.252 0 0 0 4.343,3.908a21.9,21.9 0 0 0 6.52,2.164a63.014,63.014 0 0 0 7.6,0.584c0.336,0.011 0.49,0 0.748,0c0.344,-0.034 0.445,-0.274 0.39,-0.578s-0.175,-0.683 -0.307,-1.166a50.109,50.109 0 0 0 -0.977,-3.104z"/><path fill="#8dc540" d="m26.022,13.82306c-0.022,-0.1 -0.122,-0.552 -0.223,-0.917a27.341,27.341 0 0 0 -2.94,-7.114a12.05,12.05 0 0 0 -3.3,-3.668a5.993,5.993 0 0 0 -4.932,-1.16a5.388,5.388 0 0 0 -4.194,4a5.6,5.6 0 0 0 0.422,4.031a8.2,8.2 0 0 0 4.094,3.923a17.754,17.754 0 0 0 6.561,1.525a38.759,38.759 0 0 0 4.1,0.018a0.48,0.48 0 0 0 0.412,-0.643"/></g></svg>
```

The border opacity needs to be `.07` - there is a slight difference compared to other icons

[video demonstration](https://screenshot.click/30-21-h1k9t-myz6r.mp4)

**Review by tauthomas01 — CHANGES_REQUESTED** [review]

**Comment by jiroquijano** [comment]
hello @tauthomas01, I have adjusted the opacity to .07
can you please take a look if this should be okay? Thanks.

**Review by tauthomas01 — APPROVED** [review]
Looks good. Thank you for contributing

**Pull request merged**